### DISCUSSION TITLE
Working with academic literature

### DISCUSSION THREAD

**Discussion by mada34** [BryanJenksCommunity/FAQ#30] [Q&A] [answered]
Hey Bryan: thank you for this awesome content! Yours is the first on all things Obsidian that actually makes any sense to me.

I'm a social science researcher/PhD candidate, and am struggling with applying some of these Obsidian practices to my research work. I don't have a coherent workflow at the moment, and turned to Obsidian as a way to track, consolidate, and link disparate information. I've been frustrated for years as i read and marked up papers, took notes and quotes in separate docs, and then was unable to use those messy separated info stores.

I'm curious if you can explain how you'd go about reading and taking notes on an academic paper or book. I'm just starting to understand the concept of lit notes, but feel like I'm missing something, as the system I'm seeing involves a bit of duplicated effort and a messy output: highlighting, pulling quotes into a page, putting notes onto the same page, pulling some of those notes into discrete literature notes, etc.

**Comment by tallguyjenks** [comment, marked as answer]
I have some videos that explain my direct process of paper processing, and even how to set up the software applications and tools. For books its much the same but with multiple iterations of the workflow dependent on the length/complexity of the material:

- [[[Complete Walkthrough]] Zotero & iPad Workflow For Zettelkasten Literature Notes](https://youtu.be/e7novaC_O_Y)
- [Comprehensive Obsidian & Zotero Workflow For Zettelkasten & Evergreen Notes 📝](https://youtu.be/_Fjhad-Z61o)

:)

**Reply by chass-ns** [reply]
Hi Bryan,
I followed your walk-through video in "Comprehensive Obsidian & Zotero Workflow For Zettelkasten & Evergreen Notes 📝", and it turns out that it creates span instead of links.

![image](https://user-images.githubusercontent.com/68743112/170645971-29df7991-9411-4594-ac8b-2e38966278e7.png)

How could these can be fixed?

**Reply by tallguyjenks** [reply]
might be caused by the new zotero update, currently researching
